components/tabs/skin.css

```diff
@@ -11,76 +11,198 @@ governing permissions and limitations under the License.
+  
+      .spectrum-Icon {
+        color: var(--spectrum-tabs-emphasized-texticon-tabitem-icon-color-selected);
+      }
     }
   }
 }
 
 .spectrum-Tabs--quiet {
-  border-block-end-color: var(--spectrum-tabs-m-quiet-divider-color);
+  border-block-end-color: var(--spectrum-tabs-quiet-textonly-divider-background-color);
 
   .spectrum-Tabs-selectionIndicator {
-    background-color: var(--spectrum-tabs-m-quiet-selection-indicator-color);
+    background-color: var(--spectrum-tabs-quiet-textonly-tabitem-selection-indicator-background-color-selected);
+  }
+
+  &.spectrum-Tabs--emphasized {
+    .spectrum-Tabs-selectionIndicator {
+      background-color: var(--spectrum-tabs-emphasized-texticon-tabitem-selection-indicator-background-color-selected);
+    }
   }
 }
 
 .spectrum-Tabs--vertical {
   &.spectrum-Tabs--quiet,
   &.spectrum-Tabs--compact {
-    border-inline-start-color: var(--spectrum-tabs-m-vertical-quiet-divider-color);
+    border-inline-start-color: var(--spectrum-tabs-vertical-quiet-textonly-divider-background-color);
 
     .spectrum-Tabs-selectionIndicator {
-      background-color: var(--spectrum-tabs-m-quiet-selection-indicator-color);
+      background-color: var(--spectrum-tabs-vertical-textonly-tabitem-selection-indicator-background-color-selected);
+    }
+  }
+
+  &.spectrum-Tabs--emphasized {
+    border-inline-start-color: var(--spectrum-tabs-emphasized-textonly-divider-background-color);
+    .spectrum-Tabs-selectionIndicator {
+      background-color: var(--spectrum-tabs-emphasized-texticon-tabitem-selection-indicator-background-color-selected);
+    }
+  }
+}
+
+@media (forced-colors: active) {
+  .spectrum-Tabs {
+    --spectrum-tabs-m-textonly-tabitem-focus-ring-border-color: CanvasText;
+    --spectrum-tabs-m-textonly-tabitem-icon-color-disabled: GrayText;
+    --spectrum-tabs-m-textonly-tabitem-icon-color-hover: ButtonText;
+    --spectrum-tabs-m-textonly-tabitem-icon-color-key-focus: ButtonText;
+    --spectrum-tabs-m-textonly-tabitem-icon-color-selected: ButtonText;
+    --spectrum-tabs-m-emphasized-textonly-tabitem-icon-color-selected: ButtonText;
+    --spectrum-tabs-m-textonly-tabitem-icon-color: ButtonText;
+    --spectrum-tabs-m-textonly-divider-background-color: transparent;
+    --spectrum-tabs-m-textonly-tabitem-selection-indicator-background-color: transparent;
+    --spectrum-tabs-m-textonly-tabitem-text-color-disabled: GrayText;
+    --spectrum-tabs-m-textonly-tabitem-text-color-hover: ButtonText;
+    --spectrum-tabs-m-textonly-tabitem-text-color-key-focus: ButtonText;
+    --spectrum-tabs-m-textonly-tabitem-text-color-selected: ButtonText;
+    --spectrum-tabs-m-textonly-tabitem-text-color: ButtonText;
+    --spectrum-tabs-m-vertical-rule-color: transparent;
+    --spectrum-tabs-m-quiet-textonly-divider-background-color: transparent;
+    --spectrum-tabs-m-quiet-textonly-tabitem-selection-indicator-background-color: transparent;
```

Comment:
The variables don't exist so WHCM is not working correctly. Suggest the following:
```
    --spectrum-tabs-emphasized-texticon-tabitem-icon-color-selected: HighlightText;
    --spectrum-tabs-emphasized-texticon-tabitem-selection-indicator-background-color-selected: Highlight;
    --spectrum-tabs-emphasized-texticon-tabitem-text-color-selected: HighlightText;
    --spectrum-tabs-emphasized-textonly-divider-background-color: transparent;
    --spectrum-tabs-emphasized-textonly-tabitem-icon-color-selected: HighlightText;
    --spectrum-tabs-quiet-textonly-divider-background-color: transparent;
    --spectrum-tabs-quiet-textonly-tabitem-selection-indicator-background-color-selected: Highlight;
    --spectrum-tabs-quiet-textonly-tabitem-selection-indicator-background-color: transparent;
    --spectrum-tabs-textonly-divider-background-color: transparent;
    --spectrum-tabs-textonly-tabitem-focus-ring-border-color-key-focus: ButtonText;
    --spectrum-tabs-textonly-tabitem-icon-color-disabled: GrayText;
    --spectrum-tabs-textonly-tabitem-icon-color-hover: ButtonText;
    --spectrum-tabs-textonly-tabitem-icon-color-selected-key-focus: ButtonText;
    --spectrum-tabs-textonly-tabitem-icon-color-selected: HighlightText;
    --spectrum-tabs-textonly-tabitem-selection-indicator-background-color-selected: Highlight;
    --spectrum-tabs-textonly-tabitem-text-color-disabled: GrayText;
    --spectrum-tabs-textonly-tabitem-text-color-hover: ButtonText;
    --spectrum-tabs-textonly-tabitem-text-color-selected-key-focus: ButtonText;
    --spectrum-tabs-textonly-tabitem-text-color-selected: HighlightText;
    --spectrum-tabs-textonly-tabitem-text-color: ButtonText;
    --spectrum-tabs-vertical-quiet-textonly-divider-background-color: transparent;
    --spectrum-tabs-vertical-rule-color: transparent;
    --spectrum-tabs-vertical-textonly-divider-background-color: transparent;
    --spectrum-tabs-vertical-textonly-tabitem-selection-indicator-background-color-selected: Highlight;
  ```

Reply:
@jnurthen thanks so much for those suggestions. I just pushed a commit with those vars and took a quick look in AssistivLabs, and things look ok to me. Do you mind taking another look and letting me know if we're good to go?

Reply:
LGTM

Reply:
Thanks!